Katana/Plastic/Value.swift

```diff
@@ -145,3 +187,65 @@ public struct Value: Equatable {
     return lhs.scalable == rhs.scalable && lhs.fixed == rhs.fixed
   }
 }
+
+/**
+ This allows `Value` instances to be initialized by integer and floating point literals,
+ having the scalable part equal to the numeric literal, and the fixed part equal to 0.
+ */
+extension Value: ExpressibleByIntegerLiteral, ExpressibleByFloatLiteral {
```

Comment:
👍 let's limit it to this specific use case as you propose then, it's probably the safest and more useful thing to do.

Besides adding the `+` and `-` operators with `Int` and `CGFloat`/`Double` versions, do you prefer we keep also the ones accepting `Value`s (in case I already have a `Value` instance I want to use as a offset – which would be rare) or we remove them?